.github/workflows/generator_generic_slsa3.yml

```diff
@@ -28,29 +28,34 @@ env:
   VERIFIER_RELEASE_BINARY_SHA256: 60c91c9d5b9a059e37ac46da316f20c81da335b5d00e1f74d03dd50f819694bd
   VERIFIER_RELEASE: v0.0.1
 
-###################################################################
-#                                                                 #
-#            Input and output argument definitions                #
-#                                                                 #
-###################################################################
 on:
   workflow_call:
     inputs:
       base64-subjects:
         description: "Artifacts for which to generate provenance, formatted the same as the output of sha256sum (SHA256 NAME\\n[...]) and base64 encoded."
         required: true
         type: string
+      upload-to-release:
+        description: "If true provenance is uploaded to a GitHub release for new tags."
+        required: false
+        type: boolean
+        default: true
```

Comment:
I'm not entirely convinced that we should make this the default behavior since users of the generic workflow will need to release their artifact binaries as well. It works ok if you [use the same softprops/action-gh-release](https://github.com/ianlewis/actions-test/blob/d3be73012eec38ca11d180b7e6b4330a61e0728d/.github/workflows/provenance-only.yml#L60-L81) but even then if users provide any [inputs](https://github.com/softprops/action-gh-release#inputs) to it like `name` or `tag_name` it's not going to work as expected.